tests/Resources/ODS.robot

```diff
@@ -306,6 +306,22 @@ Fetch Project Pods Info
     @{project_pods_info}=    Oc Get    kind=Pod    api_version=v1    namespace=${project}
     [Return]    @{project_pods_info}
 
+Fetch ODS Cluster Environment
+    [Documentation]  Fetches the environment type of the cluster
+    ...        project: Project name
+    ...    Returns:
+    ...        Cluster Environment (str)
+   ${match} =    Fetch Cluster Platform Type
+   Run Keyword Unless    "${match}" == 'AWS'    FAIL
```

Comment:
- Better IF ELSE (Run Keyword Unless is deprecated in RF5)
- Fail with msg=This keyword should be used only in OSD clusters

Reply:
Done